helm/hpcc/templates/_helpers.tpl

```diff
@@ -2086,3 +2086,15 @@ Pass in dict with root
  {{ printf "**** %s: %s ****" (upper $warning.severity) $warning.msg }}
 {{- end -}}
 {{- end -}}
+
+{{- define "hpcc.hasResources" -}}
+{{/*
```

Comment:
trivial: normally these comments are declared before the function.

it doesn't matter apart from consistency.
But means white space chomping may be more important, i.e. shoul be {{- /* .... */ -}}